sway-core/src/language/ty/declaration/declaration.rs

```diff
@@ -501,9 +500,7 @@ impl CollectTypesMetadata for TyDecl {
                     return Ok(vec![]);
                 }
             }
-            TyDecl::ConstGenericDecl(_) => {
-                todo!("Will be implemented by https://github.com/FuelLabs/sway/issues/6860")
-            }
+            TyDecl::ConstGenericDecl(_) => return Ok(vec![]),
             TyDecl::ErrorRecovery(..)
```

Comment:
Why not grouped with the rest of the decls that all result in `vec![]`?
```suggestion
            TyDecl::ConstGenericDecl(_)
            | TyDecl::ErrorRecovery(..)
```